sw/device/lib/dif/dif_plic.h

```diff
@@ -0,0 +1,271 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PLIC_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PLIC_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+
+/**
+ * PLIC IRQ IDs enumeration.
+ *
+ * Enumeration of all PLIC interrupt source IDs. The IRQ IDs belonging to
+ * the same peripheral are guaranteed to be consecutive.
+ */
+typedef enum dif_plic_irq_src_id {
```

Comment:
Mismatch between `enum dif_plic_irq_src_id` and `dif_plic_irq_id_t`.  Just drop the `src` in the enum name.